### PR TITLE
Sessions and oAuth validations

### PR DESCRIPTION
**Pull request by hdelarosa86**

### CODE REVIEW

File: server/db/user.js [hunk 1]
```diff
@@ -9,40 +9,46 @@ const User = db.define("user", {
     defaultValue: UUIDV4
   },
   firstName: {
-    type: STRING,
-    allowNull: false,
-    validate: {
-      notEmpty: true
-    }
+    type: STRING
+    // allowNull: false,
```
Comment (deannawong) [inline server/db/user.js:13]: Why is the validating on this commented out?

**Review by deannawong — COMMENTED** [review]
Everything looks good. 
Why is the validating on the user model is currently commented out?

File: server/db/user.js (hunk 1 above)
Comment (hdelarosa86) [reply server/db/user.js:13]: Just needed to turn off validations to get the oAuth up and running. I'll have to create a default user to pass to oAuth to pass through the User model validation

Comment (hdelarosa86) [reply server/db/user.js:13]: You can uncomment it but don't use oAuth login yet

**Review by deannawong — APPROVED** [review]
Approved

**Pull request merged**